contracts/base/DomainOwnershipManager.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
for all project 
it is a little bit complicated logic for separating contract structure
// - 1 point